source/OptimaJet.DWKit.StarterApplication/wwwroot/locales/en-us.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please move this to `source/locales`:

>I am thinking that `wwwroot/locales` is a not a good place for those messages to be checked into source.  That is DWKit source and I normally just replace the whole directory.  It will make it more difficult for DWKit upgrades.

Reply:
Changed to source/locales